src/Aquifer.API/Endpoints/Resources/Content/AssignedToSelf/List/Endpoint.cs

```diff
@@ -1,90 +1,133 @@
+    IResourceContentSearchService _resourceContentSearchService,
+    ICachingLanguageService _cachingLanguageService)
+    : Endpoint<Request, IReadOnlyList<Response>>
 {
-    /// -------------------------------------------------------------------------------------------
-    /// -------------------------------------------------------------------------------------------
-    /// IMPORTANT: WHEN THIS QUERY IS UPDATED THE /next-up ENDPOINT NEEDS TO BE UPDATED ACCORDINGLY
-    /// -------------------------------------------------------------------------------------------
-    /// -------------------------------------------------------------------------------------------
-    private const string Query = """
-                                 SELECT RCV.ResourceContentId AS ResourceContentId, RCV.Id AS ResourceContentVersionId, R.EnglishLabel, PR.DisplayName AS ParentResourceName,
-                                        L.EnglishDisplay AS LanguageEnglishDisplay, RCV.SourceWordCount AS WordCount, RC.Status AS StatusValue,
-                                        P.Name AS ProjectName, P.ProjectedDeliveryDate, History.Created AS HistoryCreated, R.SortOrder, RC.ContentUpdated
-                                 FROM ResourceContentVersions AS RCV
-                                     INNER JOIN ResourceContents AS RC ON RCV.ResourceContentId = RC.Id
-                                     INNER JOIN Resources AS R ON RC.ResourceId = R.Id
-                                     INNER JOIN ParentResources AS PR ON R.ParentResourceId = PR.Id
-                                     INNER JOIN Languages AS L ON RC.LanguageId = L.Id
-                                     LEFT JOIN ProjectResourceContents PRC ON RC.Id = PRC.ResourceContentId
-                                     LEFT JOIN Projects P ON PRC.ProjectId = P.Id
-                                     CROSS APPLY (
-                                         SELECT TOP 1 RCVAUH.Created AS Created
-                                         FROM ResourceContentVersionAssignedUserHistory AS RCVAUH
-                                         WHERE RCV.Id = RCVAUH.ResourceContentVersionId AND RCVAUH.AssignedUserId = {0}
-                                         ORDER BY RCVAUH.Id DESC
-                                     ) AS History
-                                 WHERE RCV.AssignedUserId = {0} AND RC.Status NOT IN ({1}, {2})
-                                 """;
-
     public override void Configure()
     {
         Get("/resources/content/assigned-to-self");
         Permissions(PermissionName.ReadResources);
     }
 
-    public override async Task HandleAsync(CancellationToken ct)
+    /// ---------------------------------------------------------------------------------------------------
+    /// ---------------------------------------------------------------------------------------------------
+    /// IMPORTANT: WHEN THIS SEARCH FILTER IS UPDATED THE /next-up ENDPOINT NEEDS TO BE UPDATED ACCORDINGLY
```

Comment:
I did not update the `next-up` endpoint because the results returned from the search service should match what was previously returned for this endpoint.